tom/bot.py

```diff
@@ -304,13 +305,24 @@ def find_reviewers(self, pr):
             pr.reviewers = self.default_maintainers
         pr.reviewer = random.choice(pr.reviewers)
 
+    def assign_dependabot_maintainer(self, pr):
+        if pr.author != "dependabot[bot]":
+            return
+        if pr.repo not in self.repos_dependabot:
+            log.warning(f"A dependabot PR in {pr.repo} with no assigned maintainer!")
+            return
+
+        pr.reviewer = self.repos_dependabot[pr.repo]
+
     def handle_pr(self, pr):
-        url = pr["url"].replace("https://api.github.com/repos/", "")
         log.info("Looking at: {} ({})".format(pr["title"], pr["html_url"]))
 
         pr = PR(pr, self.github)
         if "ping_reviewers" in self.bot_features:
             self.find_reviewers(pr)
+        if "ping_reviewers_dependabot" in self.bot_features:
+            self.assign_dependabot_maintainer(pr)
+        if "ping_reviewers" in self.bot_features or "ping_reviewers_dependabot" in self.bot_features:
             self.ping_reviewer(pr)
```

Comment:
So, as I pointed out face to face: You should set a default vaue for `reviewer` in `class PR` (`github.py`). Then in `ping_reviewer()` you should add an if to skip if there is no reviewer found.

Reply:
Thanks. I fixed that now.